### PR TITLE
docs: add draft for development guide

### PR DESCRIPTION
**Pull request by Tommytrg**
Add a first draft for witnet development docs.

### CODE REVIEW

**Comment by csmoe** [comment]
consider adding rust 2018 edition guide https://rust-lang-nursery.github.io/edition-guide/introduction.html

File: docs/development.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Development
+
+## Installing
+
+1. Install dependencies
+    - Rust
+    - Rocks DB ?
```
Comment (aesedepece) [inline docs/development.md]: Rocks DB is compiled by the `rocksdb-sys` crate.

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - FlatBuffers ?
```
Comment (aesedepece) [inline docs/development.md]: @mmartinbar @mariocao please shout out :wink:

Comment (mariocao) [reply docs/development.md]: `Flatbuffers` is a dependency in `data-structures` and it is not necessarily a prerequisite for developers. `Flatbuffers` may be required only if devs want to compile and re-generate the `fbs` files.

**Pull request merged**